packages/forms-web-app/src/controllers/appeals/your-appeals.js

```diff
@@ -1,17 +1,23 @@
-const { getUserAppealsById, getUserByEmail } = require('../../lib/appeals-api-wrapper');
 const { mapToAppellantDashboardDisplayData } = require('../../lib/dashboard-functions');
 const { VIEW } = require('../../lib/views');
 const logger = require('../../lib/logger');
+const { AppealsApiClient } = require('@pins/common/src/client/appeals-api-client');
+const config = require('../../config');
 
 exports.get = async (req, res) => {
-	const email = req.session.appeal?.email;
+	const apiClient = new AppealsApiClient(config.appeals.url, config.appeals.timeout);
```

Comment:
You can import this from `src/lib/appeals-api-client.js` which contains a shared instance of AppealsApiClient